### PR TITLE
Bg

### PR DESCRIPTION
**Pull request by surendrajngp**

### CODE REVIEW

File: src/homepage/home.css [hunk 1]
```diff
@@ -1,33 +1,42 @@
 @import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@500&family=Roboto+Mono:wght@600&display=swap");
 
+.homeCompo {
+  background: linear-gradient(to bottom, #f6f9ff, #f6f9ff) !important;
+}
+
```
Comment (AnmolkumarShah) [inline src/homepage/home.css:6]: not necessary, the topic is off now

File: src/homepage/homepage.jsx [hunk 2]
```diff
@@ -2,32 +2,42 @@ import React from "react";
...
+                  <b style={{ color: "#0A4158" }}>Dat</b>
+                  <b style={{ color: "#4B8378" }}>Algo</b>
+                </h1>
+                <h3 className="pt-2">
+                  A tool to Visualize Data Structures & Algorithm
+                </h3>
+                <p className="text-danger mb-0 mt-2"># Learn Anytime, Anywhere</p>
+                <p className="text-danger m-0">
+                  # We believe everyone has the capacity to be creative.
+                </p>
```
Comment (AnmolkumarShah) [inline src/homepage/homepage.jsx:34]: not require here, they will show up in about us

Comment (surendrajngp) [reply src/homepage/homepage.jsx:34]: ok

**Comment by AnmolkumarShah** [comment]
please review the changes and correct them

**Pull request closed without merging**